src/collections.ts

```diff
@@ -0,0 +1,147 @@
+import * as collections from "@opencreek/deno-std-collections"
```

Comment:
General comment: with this approach, all JSDocs for the functions are lost. This might be ok for now, but if this will be used by others, I am not too sure about it. If it is low effort to copy that over, maybe it is worth to do so?

Reply:
Will do this at a later time. Would have to change the jsdoc anyways